README.md

```diff
@@ -32,3 +32,36 @@ Use the flag `--launch` to start recording on app launch. When you first connect
 ## Under the hood
 
 ETTrace spawns a new thread which captures a stacktrace of the main thread periodically. This is sampling based profiling. The sampling thread starts either when ettrace.framework is loaded (+load method), or when the CLI sends a message to the application. These control messages and the sampled data are communicated using PeerTalk.
+
+## Adding events to the flamegraph
+
+ETTrace supports displaying events in the flamegraph from version v1.0. You can use them to track user flows in your code, you can easily add them by posting a notification in your code:
```

Comment:
flamegraph -> flamechart - Only flamecharts support events because they have time in-order on the xaxis.